king_phisher/client/tabs/mail.py

```diff
@@ -847,6 +847,37 @@ def signal_button_clicked_verify(self, button):
 			gui_utilities.show_dialog_info('Successfully Opened The Web Server URL', self.parent)
 		return
 
+	def signal_button_clicked_verify_spf(self, button):
+		sender_email = self.gobjects['entry_source_email_smtp'].get_text()
+		error_description = None
+
+		if not utilities.is_valid_email_address(sender_email):
+			gui_utilities.show_dialog_info('WARNING: Can not check SPF records for an invalid source email address.\n', self.parent)
```

Comment:
This should use show_dialog_warning instead so the WARNING prefix isn't necessary.